### PR TITLE
Update and re-org

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Review by jeff-1amstudios — APPROVED** [review]

**Comment by jeff-1amstudios** [comment]
Adding @sundippatel as a reviewer also

File: modules/ROOT/pages/manage-proxy.adoc [hunk 1]
```diff
@@ -3,12 +3,32 @@ ifndef::env-site,env-github[]
...
 
 The rtftl tool is required to manage proxy settings on Runtime Fabric. Follow the steps to xref:install-rtfctl.adoc[install rtfctl] before moving forward with configuring proxy settings.
 
+== Apply Proxy Values to Runtime Fabric Services
+
+. Log in to a controller node where `rtfctl` has been installed. 
+. Run the following command, replacing the placeholder values with the following:
++
+----
+rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.1.1.1:8888,2.2.2.2.:9999>"
```
Comment (sundippatel) [inline modules/ROOT/pages/manage-proxy.adoc]: remove trailing "." after 2.2.2.2.

File: modules/ROOT/pages/manage-proxy.adoc [hunk 2]
```diff
@@ -3,12 +3,32 @@ ifndef::env-site,env-github[]
...
+----
++
+* *<user>*: the username needed to authenticate to the HTTP proxy.
+* *<pass>*: the password needed to authenticate to the HTTP proxy. +
+[NOTE]
+The password must be URL encoded.
+* *<10.0.0.1>*: the IP address or hostname to access the HTTP proxy.
+* *<8080>*: the port on the host where the HTTP proxy is listening for requests.
+* *<1.1.1.1:8888,2.2.2.2.:9999>*: the `NO_PROXY` hosts and ports, delimited by commas.
```
Comment (sundippatel) [inline modules/ROOT/pages/manage-proxy.adoc]: remove trailing "." after 2.2.2.2

**Comment by sundippatel** [comment]
I think we made a critical error in the re-org of these steps. Looking at the changes, I now see 3 sections for proxies:
 - Apply proxy values
 - Apply monitoring proxy 
 - Apply HTTP proxy

The first and third are actually related, which is what we had in the previous version. Could we bring that back to this version?

File: modules/ROOT/pages/manage-proxy.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/manage-proxy.adoc]: Got it - thank you!

File: modules/ROOT/pages/manage-proxy.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/manage-proxy.adoc]: Got it - thank you!

**Review by sundippatel — DISMISSED** [review]
Minor re-orging and tweaking

File: modules/ROOT/pages/manage-proxy.adoc [hunk 3]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Update the proxy settings used by Runtime Fabric outbound connections to the internet. This includes managing HTTP proxies for connecting out to the Anypoint control plane, and SOCKS5 proxies for sending metrics and logs out to Anypoint Monitoring.
+You can update the proxy settings used by Runtime Fabric outbound connections to the internet. This includes managing HTTP proxies for connecting to the Anypoint control plane, and SOCKS5 proxies for sending metrics and logs to Anypoint Monitoring.
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-proxy.adoc]:
```suggestion
You can update the proxy settings used by Anypoint Runtime Fabric outbound connections to the internet. This includes managing both HTTP proxies for connecting to the Anypoint Platform control plane and SOCKS5 proxies for sending metrics and logs to Anypoint Monitoring.
```

File: modules/ROOT/pages/manage-proxy.adoc [hunk 4]
```diff
@@ -31,29 +31,18 @@ The output should match the value expected.
...
-
-. Copy the `set_dockerd_proxy.sh` from the controller node used to install Runtime Fabric, which is located under `/opt/anypoint/runtimefabric/scripts/`, to each controller and worker node under the same location (`/opt/anypoint/runtimefabric/scripts/`). +
-[NOTE]
-If you are not able to locate this script, please open a support ticket for assistance.
-. Update the `/opt/anypoint/runtimefabric/env` file to set the values for `RTF_HTTP_PROXY` and `RTF_NO_PROXY` variables. + 
-The `RTF_HTTP_PROXY` value accepts authentication values in-line. For example: `http://user:Password@1.1.1.1:80` +
-The `RTF_NO_PROXY` value accepts a comma-separated list of hosts that should not go through the proxy. For example: `1.1.1.1,no-proxy.com`
-. Run the script `/opt/anypoint/runtimefabric/scripts/set_dockerd_proxy.sh`.
-. After repeating these steps on each controller and worker node, apply these proxy values to the services running within Runtime Fabric.
+* If you are using Runtime Fabric installer 1.0.x, run the script on each node of your Runtime Fabric to update the cluster.
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-proxy.adoc:37]: Is the installer 1.0.x or is Runtime Fabric 1.0.x?

Comment (kbostrom) [reply modules/ROOT/pages/manage-proxy.adoc:37]: if the latter:
```suggestion
* If you are using Runtime Fabric 1.0.x installer, run the script on each node of your Runtime Fabric installation to update the cluster.
```
...need to give "your" an object.

File: modules/ROOT/pages/manage-proxy.adoc [hunk 5]
```diff
@@ -62,11 +51,23 @@ rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.
 The password must be URL encoded.
 * *<10.0.0.1>*: the IP address or hostname to access the HTTP proxy.
 * *<8080>*: the port on the host where the HTTP proxy is listening for requests.
-* *<1.1.1.1:8888,2.2.2.2.:9999>*: the `NO_PROXY` hosts and ports, delimited by commas.
+* *<1.1.1.1:8888,2.2.2.2:9999>*: the `NO_PROXY` hosts and ports, delimited by commas.
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-proxy.adoc:54]: When we update, let's fix list format.

File: modules/ROOT/pages/manage-proxy.adoc [hunk 6]
```diff
@@ -62,11 +51,23 @@ rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.
...
 . To verify the change was successful, run the following command to output the current value of the HTTP proxy: `rtfctl get http-proxy`.
 
 The output should match the value expected.
 
+=== Run the Script on Each Node
+
+Perform the following steps *only* if you are using Runtime Fabric installer 1.0.x. Run `rtfctl version` to verify, reported as the *Server version*.
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-proxy.adoc]: ditto question about version number
```suggestion
Perform the following steps _only_ if you are using Runtime Fabric installer 1.0.x. To verify your version, run `rtfctl version` and look for the results for *Server version*.
```

Comment (kbostrom) [reply modules/ROOT/pages/manage-proxy.adoc]: I would make this a prereq....

File: modules/ROOT/pages/manage-proxy.adoc [hunk 7]
```diff
@@ -62,11 +51,23 @@ rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.
...
+
+. Copy the `set_dockerd_proxy.sh` from the controller node used to install Runtime Fabric, which is located under `/opt/anypoint/runtimefabric/scripts/`, to each controller and worker node under the same location (`/opt/anypoint/runtimefabric/scripts/`). +
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-proxy.adoc]:
```suggestion
. From the controller node used to install Runtime Fabric, copy the `set_dockerd_proxy.sh` value located in `/opt/anypoint/runtimefabric/scripts/` to each controller and worker node in the same location (`/opt/anypoint/runtimefabric/scripts/`). +
```

Comment (kbostrom) [reply modules/ROOT/pages/manage-proxy.adoc]: Consider moving the NOTE to be just a plain sentence under the previous step.

File: modules/ROOT/pages/manage-proxy.adoc [hunk 8]
```diff
@@ -62,11 +51,23 @@ rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.
...
+[NOTE]
+If you are not able to locate this script, open a support ticket for assistance.
+. Update the `/opt/anypoint/runtimefabric/env` file to set the values for `RTF_HTTP_PROXY` and `RTF_NO_PROXY` variables. +
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-proxy.adoc]:
```suggestion
. Update the `/opt/anypoint/runtimefabric/env` file with values for both the `RTF_HTTP_PROXY` and `RTF_NO_PROXY` variables. + 
```

File: modules/ROOT/pages/manage-proxy.adoc [hunk 9]
```diff
@@ -62,11 +51,23 @@ rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.
...
 
+=== Run the Script on Each Node
+
+Perform the following steps *only* if you are using Runtime Fabric installer 1.0.x. Run `rtfctl version` to verify, reported as the *Server version*.
+
+. Copy the `set_dockerd_proxy.sh` from the controller node used to install Runtime Fabric, which is located under `/opt/anypoint/runtimefabric/scripts/`, to each controller and worker node under the same location (`/opt/anypoint/runtimefabric/scripts/`). +
+[NOTE]
+If you are not able to locate this script, open a support ticket for assistance.
+. Update the `/opt/anypoint/runtimefabric/env` file to set the values for `RTF_HTTP_PROXY` and `RTF_NO_PROXY` variables. + 
+The `RTF_HTTP_PROXY` value accepts authentication values in-line. For example: `http://user:Password@1.1.1.1:80` +
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-proxy.adoc]:
```suggestion
The `RTF_HTTP_PROXY` value accepts authentication values inline: for example, `http://user:Password@1.1.1.1:80`. +
```

File: modules/ROOT/pages/manage-proxy.adoc [hunk 10]
```diff
@@ -62,11 +51,23 @@ rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.
...
+The `RTF_NO_PROXY` value accepts a comma-separated list of hosts that should not go through the proxy. For example: `1.1.1.1,no-proxy.com`
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-proxy.adoc]:
```suggestion
The `RTF_NO_PROXY` value accepts a comma-separated list of hosts that should not go through the proxy: for example, `1.1.1.1,no-proxy.com`.
```

**Review by kbostrom — APPROVED** [review]
Done!

**Pull request merged**